CONTRIBUTING.adoc

```diff
@@ -612,25 +612,82 @@ vector: '/images/logos/${imageId}/${imageId}.svg'
 credit: 'Your Name'
 credit_url: 'https://twitter.com/yourtwitteraccount'
 ```
+== Maintainer guide
+
+This section contains information for contributors who are interested to help with the Jenkins website maintenance.
+
+=== Website structure and ownership
+
+The Jenkins website hosts various content: user and developer documentation, blog, governance materials, pages for special interest groups and sub-projects, etc.
+This information is maintained by multiple teams.
+Ownership domains are also defined in the link:/.github/CODEOWNERS[CODEOWNERS] file.
+Note that this file might be out of date or missing some entries, so common sense there applies.
+
+Notable special areas:
+
+* link:https://jenkins.io/project[Governance documents] - Managed by the link:https://www.jenkins.io/project/board/#current-board-members[Governance Board]
+* Jenkins core changelogs - Managed by the link:https://github.com/jenkinsci/jenkins/blob/master/docs/MAINTAINERS.adoc#roles[Jenkins core maintainers]
+* Security pages and advisories under `jenkins.io/security/` - Managed by the link:https://www.jenkins.io/security/team/[Jenkins Security Team]
+* Event pages under `jenkins.io/events/` - Managed by event organizers, link:https://www.jenkins.io/sigs/advocacy-and-outreach/[Advocacy and Outreach SIG] and the link:https://www.jenkins.io/project/team-leads/#events[Jenkins Events Officer]
+* SIG and sub-project pages - Managed by teams
+* etc.
```

Comment:
Doesn't really make sense given the next sentence.

Reply:
```suggestion
* etc.
```

These were examples anyway 🤷‍♂️